test/new-e2e/tests/installer/upgrade_scenario_test.go

```diff
@@ -188,6 +194,34 @@ func (s *upgradeScenarioSuite) TestStopWithoutExperiment() {
 	require.Equal(s.T(), beforeStatus, afterStatus)
 }
 
+func (s *upgradeScenarioSuite) TestConcurrentExperiments() {
+	s.RunInstallScript("DD_REMOTE_UPDATES=true")
+	defer s.Purge()
+	s.host.WaitForUnitActive(
+		"datadog-agent.service",
+		"datadog-agent-trace.service",
+		"datadog-agent-process.service",
+		"datadog-installer.service",
+	)
+
+	s.setCatalog(testCatalog)
+
+	timestamp := s.host.LastJournaldTimestamp()
+	_, err := s.startExperimentCommand(latestAgentImageVersion)
+	require.NoError(s.T(), err)
+	s.assertSuccessfulStartExperiment(timestamp, latestAgentImageVersion)
+
+	// Start a second experiment that overrides the first one
+	_, err = s.startExperimentCommand(oldAgentVersion)
+	require.NoError(s.T(), err)
+
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏Let's assert the installer status here as well

Reply:
The `s.assertSuccessful[Start|Stop]Experiment` assert the installer status
We need to assert what's the status after the 2nd start_experiment